app/javascript/packages/countdown/countdown-alert-element.spec.ts

```diff
@@ -7,7 +7,7 @@ describe('CountdownAlertElement', () => {
 
   function createElement({ showAtRemaining }: { showAtRemaining?: number } = {}) {
     document.body.innerHTML = `
-      <lg-countdown-alert ${showAtRemaining ? `show-at-remaining="${showAtRemaining}"` : ''}>
+      <lg-countdown-alert ${showAtRemaining ? `show-at-remaining="${showAtRemaining}"` : ''} >
```

Comment:
Minor, is the space before the '>' conventional?

Reply:
My mistake. Addressed this and your above comment in [64f1126](https://github.com/18F/identity-idp/pull/11899/commits/64f1126019e37c542734e9ff6e31d7926416368a)